src/EditorFeatures/Core/Implementation/ChangeSignature/AbstractChangeSignatureCommandHandler.cs

```diff
@@ -37,8 +37,8 @@ private static VSCommanding.CommandState GetCommandState(ITextBuffer subjectBuff
                 return VSCommanding.CommandState.Unspecified;
             }
 
-            var supportsFeatureService = document.Project.Solution.Workspace.Services.GetService<IDocumentSupportsFeatureService>();
-            if (!supportsFeatureService.SupportsRefactorings(document))
+            var supportsFeatureService = document.Project.Solution.Workspace.Services.GetService<ITextBufferSupportsFeatureService>();
```

Comment:
>supportsFeatureService [](start = 16, length = 22)

Is it possible for the service to be null? Should we make it a required service and call GetRequiredService?